assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/applicationContext-spring-security-oauth.properties

```diff
@@ -0,0 +1,20 @@
+shouldPerformLiveUpdate=true
+
+# OIDC Configuration for Azure AD
+azure.id=azure
+azure.client-name=azure
+azure.client-id=8ca223e9-fc41-4751-8dc3-7b1fbdabcbb3
+azure.client-secret=Anything
```

Comment:
This needs to be encrypted at rest. It cannot be plain-text.

Reply:
Not pushing the encoded string as of now. 
Later, this file will be masked since client has to enter their own OAuth identity provider details.

Reply:
Customers will need to enter an encrypted value here, which will need to be decrypted with the pentaho-encryption module in your code later.